### DISCUSSION TITLE
Laser grbl  with Chinese clone V4 shield.

### DISCUSSION THREAD

**Discussion by Dave-v-price** [arkypita/LaserGRBL#1554] [Q&A]
Hi 
Does anyone know if the laser grbl nano firmware. Reconfigured the XYZ step and direction pins     To fix the wrong pin wiring on the Chinese V4 shield.   
the default grbl configuration for uno v3 uses pins 234 for  XYZ step and 567 for XYZ direction.
The v4 shield has pins 567 for XYZ step and 123 for Direction.
 If it doesn’t how can I reconfigure the firmware files which are in a hex format.

**Comment by svenhb** [comment]
I did it - switching dir and step signals, read here: http://svenhb.bplaced.net/?CNC___Klappbarer_Plotter
You will find a translation button on the left side menu.
The hex can be downloaded here: https://github.com/svenhb/GRBL-Plotter/tree/master/Firmware/hex_files

**Reply by Dave-v-price** [reply]
Hi Sven.
   Thanks for your reply and link.
I used your modified hex file with LaserGrbl. I now get smooth control of the XY axis.
